### PR TITLE
fix(university-gateway): consolidate infra service name

### PR DESCRIPTION
**Pull request by busla**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/services/university-gateway/infra/university-gateway.ts [hunk 1]
```diff
@@ -5,110 +5,109 @@ import {
...
+const namespace = serviceName
+const imageName = serviceName
 
 const postgresInfo = {
-  username: 'university_gateway',
-  name: 'university_gateway',
-  passwordSecret: '/k8s/university-gateway/DB_PASSWORD',
+  username: dbName,
+  name: dbName,
+  passwordSecret: `/k8s/${dbName}/DB_PASSWORD`,
```
[the remark this answers is not in the document]
Comment (busla) [reply apps/services/university-gateway/infra/university-gateway.ts]: yep, well spotted 🙌🏼

File: apps/services/university-gateway/infra/university-gateway.ts [hunk 2]
```diff
@@ -5,110 +5,109 @@ import {
...
+      limits: { cpu: '200m', memory: '384Mi' },
+      requests: { cpu: '50m', memory: '256Mi' },
+    })
+    .env({
+      IDENTITY_SERVER_CLIENT_ID: '@island.is/clients/university-gateway',
+      IDENTITY_SERVER_ISSUER_URL: {
+        dev: 'https://identity-server.dev01.devland.is',
+        staging: 'https://identity-server.staging01.devland.is',
+        prod: 'https://innskra.island.is',
+      },
```
Comment (AndesKrrrrrrrrrrr) [inline apps/services/university-gateway/infra/university-gateway.ts:38]: Is this needed?

**Review by AndesKrrrrrrrrrrr — APPROVED** [review]
DevOps files LGTM

File: apps/services/university-gateway/infra/university-gateway.ts (hunk 2 above)
Comment (jonarnarbriem) [reply apps/services/university-gateway/infra/university-gateway.ts:38]: This seems to be defined in a similar way for most other services. [Endorsement System for example](https://github.com/island-is/island.is/blob/4f6728fa816c37d5720b184e757165aed9898f27/apps/services/endorsements/api/infra/endorsement-system-api.ts#L46)

**Pull request merged**